src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -306,6 +309,25 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
 
   const [wasRegisterVisited, setWasRegisterVisited] = useState(false);
 
+  useEffect(() => {
+    if (isEdit) {
+      return;
+    }
+
+    let defaultTimezone = '';
+    if (distribution === RHEL_10) {
+      defaultTimezone = 'Etc/UTC';
+    } else if (targetEnvironments.includes('azure')) {
+      defaultTimezone = 'Etc/UTC';
+    } else {
+      defaultTimezone = 'America/New_York';
+    }
+
+    if (!timezone && defaultTimezone) {
+      dispatch(changeTimezone(defaultTimezone));
+    }
+  }, [distribution, targetEnvironments, isEdit, dispatch]);
```

Comment:
can you add the missing dependency in useEffect? < timezone is the missing one :)> 
 [distribution, targetEnvironments, isEdit, timezone, dispatch]);